### ISSUE TITLE
Feature Request: Show the minutes and hours of downloaded content playtime

### ISSUE THREAD

**Issue by godalming123** [futo-org/grayjay-android#606]
It would be useful to be able to see how many minutes and or hours of playtime their is in the content that has been downloaded, to know wether the content that has been downloaded will last for a given journey.

**Comment by VoxelPrismatic** [comment]
Close as complete?

<img width="1096" height="544" alt="Image" src="https://github.com/user-attachments/assets/44e9152e-95fe-49fb-b3b5-47c64199400b" />

**Comment by godalming123** [comment]
@VoxelPrismatic, there should also be the amount of playtime for individual playlists

**Comment by VoxelPrismatic** [comment]
you mean how much is downloaded within a playlist?
![ResizedImage_2025-11-12_12-22-18_1908.png](https://github.com/user-attachments/assets/3b1000fb-e92a-4e55-9e30-79898fc93b3a)

**Comment by godalming123** [comment]
@VoxelPrismatic, yeah, I've just realised that not showing the hours and minutes of downloaded content only applies to the watch later playlist.

**Comment by defiling9046** [comment]
@Zvonimir-FUTO do you want to change the title of this issue to "Bug: Playtime not shown in the watch later playlist"

**Comment by defiling9046** [comment]
@Zvonimir-FUTO this can be closed in favour of my new bug report #3440
